src/yunohost/alias.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+
+""" License
+
+    Copyright (C) 2016 YUNOHOST.ORG
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program; if not, see http://www.gnu.org/licenses
+
+"""
+
+""" yunohost_alias.py
+
+    Manage alias
+"""
+
+import errno
+
+from moulinette.core import MoulinetteError
+
+def alias_list(auth, fields=None, filter=None, limit=None, offset=None):
+    """
+    List alias
+
+    Keyword argument:
+        filter -- LDAP filter used to search
+        offset -- Starting number for alias fetching
+        limit -- Maximum number of alias fetched
+        fields -- fields to fetch
+
+    """
+    _ensure_ou_created(auth)
+
+    alias_attrs = { 'mail': 'alias',
+                    'maildrop': 'mail-forward' }
+    attrs = []
+    result_list = []
+
+    # Set default arguments values
+    if offset is None:
+        offset = 0
+    if limit is None:
+        limit = 1000
```

Comment:
this is pure copy-paste from the user command. what's the problem with all three arguments being used together ?